serverless-map-reduce/binaris.yml

```diff
@@ -16,3 +16,23 @@ functions:
     runtime: node8
     env:
       BINARIS_ACCOUNT_ID:
+  public_amr_controller:
+    file: amr.js
+    entrypoint: controller
+    executionModel: concurrent
+    runtime: node8
+    env:
+      BINARIS_ACCOUNT_ID:
```

Comment:
This is always available in the environment variables of a function, no need to add.